docs/authentication.md

```diff
@@ -6,20 +6,91 @@ APIs should use the authentication methods described in the public [Authenticati
+
+If the token is not usable on your API, you should return a [403 error (see authentication docs)](https://publiq.stoplight.io/docs/authentication/docs/errors.md#forbidden). (The difference with the other checks being that the token is valid on _some_ of our APIs, but forbidden on your specific API.)
+
+The values inside the `https://publiq.be/publiq-apis` claim are the keys that are also stored in the client metadata. For example a token can have the following claims:
+
+    {
+      "sub": "auth0|...",
+      "aud": "https://api.publiq.be",
+      ...
+      "https://publiq.be/publiq-apis": [
+        "uitpas",
+        "sapi3"
+      ]
+    }
+
+Note that the claim can be missing, in which case it is considered to be empty. If it is present it can contain 0, 1, or multiple values.
+
+## Permissions
+
+User/client permissions to perform actions inside your API should be managed inside the API itself.
+
+If the specific client or user cannot perform a requested action, you should return a [403 error (see authentication docs)](https://publiq.stoplight.io/docs/authentication/docs/errors.md#forbidden).
+
 ## Scopes
 
-To determine if a client has permission to use a specific API, the API should validate the [scopes](https://publiq.stoplight.io/docs/authentication/docs/scopes.md) of the client.
+Scopes should only be used for granting access to resources on an API if those resources belong to a specific user and that user should determine if a client can access them or not.
+
+The user will see a list of permissions that they are granting the client the first time after they login on Auth0 with that specific (3rd party) client.
+
+![An example screenshot of the screen that the user will see after logging in for the first time through a client that requests scopes.](https://images.ctfassets.net/cdy7uua7fh8z/1te4FYRbu0aFcdohdXY2Rv/116bed5515eb2114c39374fb0a258912/consent-screen.png)
+
+Suitable (fictional) examples of scopes could be:
+
+-   Reading a user's events (`events:read`)
+-   Updating a user's events (`events:update`)
+-   Deleting a user's events (`events:delete`)
+-   Reading a user's recommendations (`recommendations:read`)
+-   Reading a user's likes (`likes:read`)
+-   Updating a user's likes (`likes:update`)
+-   Reading a user's newsletter subscriptions (`newsletters:read`)
+-   Updating a user's newsletter subscriptions (`newsletters:update`)
 
-For example to access the UiTPAS API, the `https://api.publiq.be/auth/uitpas` scope is required.
+> At the time of writing, no publiq APIs actually use scopes yet for this purpose. Do not use it if the granular access is not really needed _and_ the user needs to be able to grant them themself. This section just covers the intended use of scopes, _if_ you need them.
```

Comment:
het is nu wel een beetje dubbel: we documenteren een fictief voorbeeld, maar we zeggen direct daaronder: doe dit niet...

Reply:
Ik dacht toe te lichten waarvoor scopes wél dienen, en van het enkel daar voor te gebruiken en niet voor workarounds voor audience bv zoals we vroeger deden (of mogelijks andere workarounds). Maar misschien is dit dan de verkeerde manier. Ik kan ook gewoon zeggen "never use scopes" maar dan is het misschien niet duidelijk waarom 😂

Reply:
Ik heb het proberen verduidelijken in 2c81830